comps/lvms/llava/lvm.py

```diff
@@ -28,6 +28,9 @@
 logger = CustomLogger("lvm")
 logflag = os.getenv("LOGFLAG", False)
 
+# The maximum number of images that should be sent to the LVM
+max_images = int(os.getenv("MAX_IMAGES", 1))
```

Comment:
In this line, is `1` being set manually? Or is it a default?

Reply:
This means that if `MAX_IMAGES` is unset, it will default to 1